v3/tutorials/02-proof-of-existence/index.mdx

```diff
@@ -106,74 +106,65 @@ Therefore, the first step is to remove some files and content from the files in
    tests.rs
    ```
 
-1. Open the `lib.rs` file in a text editor.
+1. Open the `pallets/template/src/lib.rs` file in a text editor.
 
    This file contains code that you can use as a template for a new pallet.
    You won't be using the template code in this tutorial.
    However, you can review the template code to see what it provides before you delete it.
 
-1. Delete all of the lines in the `lib.rs` file.
-
-1. Add the macro required to build both the native Rust binary (`std`) and the WebAssembly (`no_std`) binary.
+1. Replace all contents of `pallets/template/src/lib.rs` with the following skeleton code that includes all of the [***mandatory*** macros for FRAME V2](https://docs.substrate.io/rustdocs/latest/frame_support/attr.pallet.html):
```

Comment:
Note the macros below are NOT all mandatory. Only if you need them. For example `event`, `error`, `call`, and `storage` could all be left out.